### ISSUE TITLE
I can't understand why the accordingly labels for positives is still 0, for i think it should be 1.

### ISSUE THREAD

**Issue by buluofeng2** [nikheelpandey/TAUP#6]
In loss.py,
```
        positives = torch.cat([l_pos, r_pos]).view(2 * N, 1)    # [2N, 1]

        # print(positives)

        # get the values of every pair that's a mismatch
        diag = torch.eye(2*N, dtype=torch.bool, device=device)
        diag[N:,:N] = diag[:N,N:] = diag[:N,:N]     # 将矩阵看作四个小方块，每个方块的对角线都变成单位矩阵
        negatives = similarity_matrix[~diag].view(2*N, -1)  # [2N, 2N-1]

        
        logits = torch.cat([positives, negatives], dim=1)   # [2N, 2N]
        logits /= self.temp
        labels = torch.zeros(2*N, device=device, dtype=torch.int64)
# For the labels, i think the first line should be 1
        loss = F.cross_entropy(logits, labels, reduction='sum') # log_softmax() + NLLLoss() 
        
        return (loss / (2 * N))
```
